### PR TITLE
chore: Remove Rockset from feast

### LINKED ISSUE #4404: Remove Rockset from Feast
**Is your feature request related to a problem? Please describe.**
[Rockset was acquired by OpenAI](https://rockset.com/blog/openai-acquires-rockset/) 

As stated in their blog post:
>Existing Rockset customers will experience no immediate change. We will gradually transition current customers off Rockset and are committed to ensuring a smooth process.

So we should purge Rockset from Feast

**Describe the solution you'd like**
Deletion.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by tokoko**
Fixes #4404

### CODE REVIEW

File: sdk/python/docs/source/feast.infra.feature_servers.aws_lambda.rst [hunk 1]
```diff
@@ -1,29 +0,0 @@
-feast.infra.feature\_servers.aws\_lambda package
```
Comment (franciscojavierarceo) [inline sdk/python/docs/source/feast.infra.feature_servers.aws_lambda.rst:1]: Do we need to remove this?

File: sdk/python/docs/source/feast.infra.feature_servers.gcp_cloudrun.rst [hunk 2]
```diff
@@ -1,29 +0,0 @@
-feast.infra.feature\_servers.gcp\_cloudrun package
```
Comment (franciscojavierarceo) [inline sdk/python/docs/source/feast.infra.feature_servers.gcp_cloudrun.rst:1]: Why remove this?

File: sdk/python/docs/source/feast.infra.materialization.contrib.bytewax.rst [hunk 3]
```diff
@@ -1,29 +0,0 @@
-feast.infra.materialization.contrib.bytewax package
```
Comment (franciscojavierarceo) [inline sdk/python/docs/source/feast.infra.materialization.contrib.bytewax.rst:1]: Why remove this?

File: sdk/python/docs/source/feast.infra.materialization.lambda.rst [hunk 4]
```diff
@@ -1,29 +0,0 @@
-feast.infra.materialization.lambda package
```
Comment (franciscojavierarceo) [inline sdk/python/docs/source/feast.infra.materialization.lambda.rst:1]: Why remove this?

File: sdk/python/docs/source/feast.infra.registry.contrib.postgres.rst [hunk 5]
```diff
@@ -1,21 +0,0 @@
-feast.infra.registry.contrib.postgres package
```
Comment (franciscojavierarceo) [inline sdk/python/docs/source/feast.infra.registry.contrib.postgres.rst:1]: Why remove this?

File: sdk/python/docs/source/feast.infra.registry_stores.rst [hunk 6]
```diff
@@ -1,21 +0,0 @@
-feast.infra.registry\_stores package
```
Comment (franciscojavierarceo) [inline sdk/python/docs/source/feast.infra.registry_stores.rst:1]: Why remove this?

File: sdk/python/docs/source/feast.rst [hunk 7]
```diff
@@ -252,14 +260,6 @@ feast.proto\_json module
    :undoc-members:
    :show-inheritance:
 
-feast.prova module
```
Comment (franciscojavierarceo) [inline sdk/python/docs/source/feast.rst:255]: Why remove this?

**Review by franciscojavierarceo — CHANGES_REQUESTED** [review]
Nice! Think you may have deleted some extra stuff in some cases. Mind confirming?

**Comment by tokoko** [comment]
I simply regenerated sphinx docs. Those files should have been deleted in previous PRs when those modules were deleted, but looks like they stuck around. I think the problem is that our sphinx regeneration command in Makefile only updates existing files, but doesn't delete already existing ones. I'll try to fix that later as well.

**Comment by franciscojavierarceo** [comment]
Got it that makes sense.

**Comment by tokoko** [comment]
@franciscojavierarceo are you waiting on any changes for this to be merged?

**Review by franciscojavierarceo — APPROVED** [review]

**Pull request merged**